docs/api/apiv3/paths/storage_folders.yml

```diff
@@ -0,0 +1,28 @@
+# /api/v3/storages/{id}/folders
+---
+post:
+  summary: Creation of a new folder
+  operationId: storage_new_folder
+  tags:
+    - File links
+  description: Creates a new folder on the given path
```

Comment:
🟡 description is still written for "path", should be mentioning a specific parent folder.